README.md

```diff
@@ -7,6 +7,56 @@ Create a community of like minded authors to foster inspiration and innovation
 by leveraging the modern web.
 
 ---
+## Application local setup
+
+### Technologies used to build the application
+1. [Django](https://www.djangoproject.com/)
+2. [Django Rest-Framework](https://www.django-rest-framework.org/)
+3. [Brew](https://brew.sh/)
+4. [PostgreSQL](https://medium.com/@Umesh_Kafle/postgresql-and-postgis-installation-in-mac-os-87fa98a6814d)
+5. [Virtualenv](https://virtualenv.pypa.io/en/latest/installation/)
+
+### Getting Started
+1. Open your terminal and clone the repo with `https://github.com/andela/ah-bird-box.git`
+2. Switch directory to the cloned project `cd ah-bird-box`
+3. Checkout to the chore branch `git checkout ch-update-readme-163588357`
```

Comment:
To Setup the app you don't need to checkout to this branch because after it's merged it will be deleted

Reply:
Noted and fixed.